packages/flutter/test/foundation/stack_trace_test.dart

```diff
@@ -5,11 +5,13 @@
 import 'package:flutter/foundation.dart';
 import 'package:test/test.dart';
 
+import 'platform_utils.dart';
+
 void main() {
   // TODO(8128): These tests and the filtering mechanism should be revisited to account for causal async stack traces.
```

Comment:
please while you're here fix this TODO to be the right syntax ("TODO(username): description, https://bug/url")